src/framework/filterControl/forms/MultipleSelectFilterForm.spec.jsx

```diff
@@ -137,8 +167,10 @@ describe('MultipleSelectFilterForm', () => {
       const filter = props.onAddFilter.calls.argsFor(0)[0];
 
       expect(filter.comparisonValue.length).toBe(2);
-      expect(filter.comparisonValue.indexOf('Active')).not.toBe(-1);
-      expect(filter.comparisonValue.indexOf('Archived')).not.toBe(-1);
+      expect(filter.comparisonValue.filter(e => e.name === 'Active').length
+        ).not.toBe(-1);
```

Comment:
What about checking the object rather than checking the content of it. Something like:
```
expect(filter.comparisonValue.length).toBe(2);
filter.comparisonValue.forEach((comparisonValueItem, index) => {
  expect(
    filter.comparisonValue[index]
  ).toBe(filterOption.comparisonParameters.oneOfOptions[index]);
});
```

And one empty like before the `it`. (Plus opening bracket on same level as closing)